src/person/actions/register-with-email-and-password.action.ts

```diff
@@ -0,0 +1,29 @@
+// import { PersonApi } from '../api/person.api';
+import { IPersonMeResponse, IPersonRegister } from '../types';
+import { Fetcher } from '../../common/api';
+import { endpoints } from '../api/endpoints';
+import { errorFactory, PersonErr } from '../errors';
+
+export const registerWithEmailAndPassword = async (
+  fetcher: Fetcher,
+  data: IPersonRegister,
+): Promise<IPersonMeResponse[]> => {
```

Comment:
Cette route ne retourne pas un tableau mais un objet
```suggestion
): Promise<IPersonMeResponse> => {
```